lib/components/UploadList.jsx

```diff
@@ -69,13 +69,19 @@ var UploadList = React.createClass({
     var showDetailsThisUpload = _.includes(this.state.showErrorDetails, upload.key);
     var errorDetails = showDetailsThisUpload ? (<div className="UploadList-error-details">{upload.error.debug}</div>) : null;
     var showErrorsText = showDetailsThisUpload ? this.props.text.HIDE_ERROR : this.props.text.SHOW_ERROR;
-    
+    var errorMessage = upload.error.driverLink ? <div className="UploadList-error-message-wrapper">
```

Comment:
Yes, I wish there was a way of determining if you're running El Capitan or Yosemite, but at the moment it'll display that error message to all Mac users if your device is not plugged in.